### PR TITLE
docs: add line length note to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tbpg**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -283,6 +283,16 @@ API call (which uses `req`) is easier to understand.
 + }
 ```
 
+## Line length
+
+Generally, Go code does not have a strict line length limit. See
+https://github.com/golang/go/wiki/CodeReviewComments#line-length. However,
```
Comment (lesv) [inline CONTRIBUTING.md:289]: `[Wiki](https://github.com/golang/go/wiki/CodeReviewComments#line-length)`

**Review by lesv — APPROVED** [review]
minor suggestion

File: CONTRIBUTING.md (hunk 1 above)
Comment (tbpg) [reply CONTRIBUTING.md]: Done.

**Review by tbpg — COMMENTED** [review]
Thanks!

**Pull request merged**